docs/getting_started.md

```diff
@@ -0,0 +1,129 @@
+ok: [localhost] => {
+    "changed": false, 
+    "msg": [
+        "Hostname: <catasb-host>", 
+        "Next steps:", 
+        "1) Visit https://apiserver-service-catalog.<catasb-host>", 
+        "2) Accept the certificate", 
+        "3) Visit https://<catasb-host>:<port> for the console", 
+        "OR", 
+        "For CLI access:", 
+        "oc login --insecure-skip-tls-verify <catasb-host>:<port> -u admin -p admin", 
+        ""
+    ]
+}
+```
+
+Test out your development environment using [apb-examples](https://github.com/fusor/apb-examples):
+
+```bash
+git clone git@github.com:fusor/apb-examples.git
+cd hello-world-apb
+apb prepare
+docker build -t <docker-user>/hello-world-apb .
+docker run -e "OPENSHIFT_TARGET=https://<catasb-host>:<port>" -e "OPENSHIFT_USER=admin" -e "OPENSHIFT_PASS=admin" <docker-user>/hello-world-apb provision
+```
+This should have deployed a hello-world-apb application to your openshift cluster.  Logging on to your cluster at https://<catasb-host>:<port>, you should be able to view the project and click on the deployed application.
+
+<a name="first-apb"></a>
+## Creating your first APB
+In this tutorial we'll create a simple PostgreSQL database.
+
+### Using apb init
+Our first task is to create the skeleton for your app using the apb tool.  The command for this is simple:
+```bash
+apb init pg-apb
+```
+At this point you should see the following file structure
+```
+pg-apb/
+├── apb.yml
+├── Dockerfile
+├── playbooks
+└── roles
+```
+Two files were created at the root directory.  An apb.yml and a Dockerfile.  These are the minimum required for any APB.  For more information, visit the [design doc](https://github.com/fusor/ansible-playbook-bundle/blob/master/docs/design.md#apb-yml).
+```yaml
+# apb.yml
+id: d05...44b
+name: sample-apb
+image: organization/sample-apb
+description: "This is a sample skeleton file generated by `apb init`."
+bindable: false
+async: optional
+parameters:
+  - name: sample_param
+    description: Sample description of APB
+    type: string
+    default: sample-param-string
+```
+```dockerfile
+# Dockerfile
+FROM ansibleplaybookbundle/apb-base
+# MAINTAINER {{ $MAINTAINER }}
+
+LABEL "com.redhat.apb.version"="0.1.0"
+LABEL "com.redhat.apb.spec"=\
+"aWQ6IGQw....mluZwo="
+
+USER apb
+```
+
+At this point we have a fully formed APB that we can build
+```bash
+cd pg-apb
+apb prepare
+docker build -t <docker-user>/pg-apb .
+docker push <docker-user>/pg-apb
+```
+
+Visiting the OpenShift console UI at https://<catasb-host>:<port> will now display the new Ansible Playbook bundle named pg-apb in the catalog under the "All" tab.  
+
+<a name="actions"></a>
+### Actions
+Unfortunately, the brand new APB created in the last section doesn't do very much.  For that, we'll have to add some actions.  The actions supported are [provision](#provision), [deprovision](#deprovision), [bind](#bind), and [unbind](#unbind).  We'll add each of these actions in the following sections
```

Comment:
I'd remove 'Unfortunately'. The brand new APB, created in the last section, doesn't do very much.